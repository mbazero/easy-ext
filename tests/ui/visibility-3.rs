mod foo {
    use easy_ext::ext;

    #[ext(StrExt)]
    impl str {
        fn bar(&self, pat: &str) -> String {
            self.replace(pat, "_")
        }
    }
}

fn main() {
    use foo::StrExt; //~ ERROR trait `StrExt` is private [E0603]
}
