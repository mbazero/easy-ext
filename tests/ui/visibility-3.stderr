error[E0603]: trait `StrExt` is private
  --> $DIR/visibility-3.rs:13:14
   |
13 |     use foo::StrExt; //~ ERROR trait `StrExt` is private [E0603]
   |              ^^^^^^
